File: Code Generation/responses/cpp/claude/easy/unique-3-digit-even-numbers.cpp
#include <vector>
#include <set>
#include <algorithm>
using namespace std;

class Solution {
public:
    int totalNumbers(vector<int> digits) {
        set<int> result;
        sort(digits.begin(), digits.end());
        int n = digits.size();
        for (int i = 0; i < n; i++)
            for (int j = 0; j < n; j++)
                for (int k = 0; k < n; k++) {
                    if (i == j || i == k || j == k) continue;
                    if (digits[i] == 0) continue;
                    if (digits[k] % 2 != 0) continue;
                    result.insert(digits[i]*100 + digits[j]*10 + digits[k]);
                }
        return result.size();
    }
};

File: Code Generation/responses/python/claude/easy/unique-3-digit-even-numbers.py
from typing import List
from itertools import permutations

class Solution:
    def totalNumbers(self, digits: List[int]) -> int:
        seen = set()
        for perm in permutations(digits, 3):
            if perm[0] != 0 and perm[2] % 2 == 0:
                seen.add(perm[0] * 100 + perm[1] * 10 + perm[2])
        return len(seen)